src/components/ReceiveTableContainer/ReceiveTable/ReceiveTable.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./ReceiveTable.module.scss";
+import { BoardView } from "./BoardView/BoardView";
+import { Header } from "./Header/Header";
+import { Board } from "common";
+import { TableUpdater } from "./TableUpdater";
+
+type Props = {
+    boards: Record<string, Board>;
+};
+
+export const ReceiveTable = ({ boards }: Props) => {
+    return (
+        <TableUpdater>
+            <div className={styles.newReceiveTable}>
+                <Header items={["ID", "NAME", "COUNT", "CYCLE (ns)"]} />
```

Comment:
Maybe its better to define the array in a const outside the html and pass the const here